src/SmartComponents/Advisor/Advisor.cy.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import Advisor from './Advisor';
+import { advisorInterceptors, featureFlagsInterceptors } from '../../../cypress/support/interceptors';
+import Masonry from 'react-masonry-css';
+import { Grid } from '@patternfly/react-core/dist/esm/layouts';
+import { BrowserRouter } from 'react-router-dom';
+import messages from '../../Messages';
+
+before(() => {
+    cy.mockWindowChrome();
+});
+
+const breakpointColumnsObj = {
+    default: 2,
+    992: 1
+};
+
+const MountAdvisorCard = () => {
+    return (
+        <BrowserRouter>
+            <Grid hasGutter>
+                <Masonry
+                    breakpointCols={breakpointColumnsObj}
+                    className="ins-l-masonry"
+                    columnClassName="ins-l-masonry_column"
+                >
+                    <Advisor />
+                </Masonry>
+            </Grid>
+        </BrowserRouter>
+    );
+};
+
+describe('Advisor component renders', () => {
+    beforeEach(() => {
+        advisorInterceptors.rules();
+        advisorInterceptors.systems();
+        advisorInterceptors.rule();
+        featureFlagsInterceptors.edgeParityDisabled();
+        cy.mountWithContext(MountAdvisorCard);
+        cy.get('div[class*="skeleton"]').should('not.exist');
+    });
+
+    it('The count is correct', () => {
+        // Advisor recommendations section
+        cy.get('span:contains("incidents detected")').invoke('text').should((text) => {
+            expect(parseInt(text.split(' ')[0].trim(), 10)).to.equal(4);
+        });
+
+        // Recommendations by total risk section
+        cy.get('a:contains("Critical")').find('span').first().invoke('text').should((text) => {
+            expect(parseInt(text.trim(), 10)).to.equal(3);
```

Comment:
@opacut I think it would be more readable if we have some constants used here `equal(3)` instead of numbers. Also we can replace this complex logic of parseInt, trim, etc with:

```
        cy.get('a:contains("Critical")')
        .find('span')
        .first()
        .should('have.text', criticalRecommendationNum);
```
and `criticalRecommendationNum` can be obtained from fixture response body